### PR TITLE
Update vulnerable packages according to npm audit and internal discussion

### PR DESCRIPTION
**Pull request by JunyiYi**
`vscode-extension-telemetry` should be at least `v0.0.17`.

### CODE REVIEW

**Comment by metacpp** [comment]
>                 "@types/node": "*"

Why did you change it?

---
Refers to: package-lock.json:26 in dd299f4. [](commit_id = dd299f46de512337bfc319785adac605efd0eab3, deletion_comment = False)

**Comment by JunyiYi** [comment]
@metacpp Because it is changed by `npm audit fix`.

File: package.json [hunk 1]
```diff
@@ -147,18 +147,18 @@
...
         "fs-extra": "^4.0.2",
-        "lodash": "^4.17.4",
+        "lodash": "^4.17.10",
         "ms-rest": "^2.3.2",
-        "ms-rest-azure": "^2.5.5",
+        "ms-rest-azure": "^2.5.7",
         "opn": "5.1.0",
         "request-promise": "4.2.1",
-        "vscode-extension-telemetry-wrapper": "0.2.2"
+        "vscode-extension-telemetry-wrapper": "0.2.4"
```
Comment (metacpp) [inline package.json]: What is this?

Comment (metacpp) [reply package.json]: Test

**Pull request merged**